tests/chapter4-base-tests.md

```diff
@@ -89,6 +89,19 @@ Attributes can be set on tags:
 	   <body>
 	     <div name1="value1" name2="value2">
 	       'text'
+		   
+Test that the italic </i> tag is parsed correctly:
+
+	>>> test_parse("<i>A</i>B")
+	 <html>
+	   <body>
+		 <i>
+		   'A'
+		 </i>
+		 'B'
+	   </body>
+	 </html>	   
```

Comment:
Our `print_tree` doesn't print close tags.

Reply:
This isn't even the test! The issue is that some students, in `close_tag`, do:

```
def close_tag(self, node):
    # ...
    if node.tag == "/i":
      # ...
```

However, `node.tag` never includes the slash, they should write `node.tag == "i"`. If they make this mistake, the `</i>` behavior is never triggered and text never stops being italics. *That's* what we want to test for: not parsing, but whether we actually stop italicizing stuff when we reach a `</i>` tag.